AY2021S2-CS2103-T14-1/tp
# Docs(UG): Update find commands and param justification

### What this does
Closes #323
Closes #365 

### How to test
- Check justification for `NAME`, `EMAIL`, `DESCRIPTION` and `TAG`
- Check `rfind`, `ofind` and `ifind`
